### PR TITLE
Add expand feature to notification drawer

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: tests/pages/_includes/widgets/notification-drawer.html [hunk 1]
```diff
@@ -88,6 +89,19 @@ <h4 class="panel-title">
         $drawer.addClass('hide');
       }
     });
+    $('.drawer-pf-toggle-expand').click(function() {
+      var $drawer = $('.drawer-pf');
+      var $drawerNotifications = $drawer.find('.drawer-pf-notification');
+
+      if ($drawer.hasClass('drawer-pf-expanded')) {
+        $drawer.removeClass('drawer-pf-expanded');
+        $drawerNotifications.removeClass('expanded-notification')
```
Comment (dgutride) [inline tests/pages/_includes/widgets/notification-drawer.html]: Closing semicolon missing here (and all of the add/removeClass lines that were added)

**Comment by jeff-phillips-18** [comment]
Added missing semi-colons ( @dgutride issue )

**Comment by dlabrecq** [comment]
LGTM

**Comment by priley86** [comment]
LGTM

**Comment by dgutride** [comment]
Looks good

**Pull request merged**